src/git/zcl_abapgit_git_porcelain.clas.abap

```diff
@@ -10,6 +10,7 @@ CLASS zcl_abapgit_git_porcelain DEFINITION
         files   TYPE zif_abapgit_definitions=>ty_files_tt,
         objects TYPE zif_abapgit_definitions=>ty_objects_tt,
         branch  TYPE zif_abapgit_definitions=>ty_sha1,
+        commit  TYPE zif_abapgit_definitions=>ty_sha1,
```

Comment:
just like mentioned a year ago(?) for the upload pack methods, the above is redundant, a branch is a commit

Reply:
Yes, that's true but then we should rename branch to a more dynamic attribute name - what do you suggest?

Reply:
yes, for this "branch" should be renamed to "commit", since the old code is misleading